src/Tokenizer/Tokens.php

```diff
@@ -734,9 +734,9 @@ public function getTokenOfKindSibling(int $index, int $direction, array $tokens
     /**
      * Get index for closest sibling token not of given kind.
      *
-     * @param int                           $index     token index
-     * @param -1|1                          $direction
-     * @param list<array{int}|string|Token> $tokens    possible tokens
+     * @param int                                   $index     token index
+     * @param -1|1                                  $direction
+     * @param list<_PhpTokenPrototypePartial|Token> $tokens    possible tokens
```

Comment:
This extends the type (accepting second array element), but I guess it is intended to avoid too many different types for the tokens?

Reply:
partial so 2nd array element optional

and yes, the method is already accepting 2-element arrays for comparison